bucket/ricochet.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "1.1.4",
+    "description": "Anonymous peer-to-peer instant messaging.",
+    "homepage": "https://github.com/ricochet-im/ricochet",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/ricochet-im/ricochet/releases/download/v1.1.4/ricochet-1.1.4-win-install.exe",
+    "hash": "9096f058b8471ca7b1204d1acb34114e497b07902eb7811fc414d52b45b7db59",
+    "depends": "tor",
+    "innosetup": true,
+    "bin": "ricochet.exe",
+    "shortcuts": [
+        [
+            "ricochet.exe",
+            "Ricochet"
+        ]
+    ],
+    "post_install": "Remove-Item \"$dir\\tor.exe\" -Force"
+}
```

Comment:
No checkver and autoupdate

Reply:
It has stoped for more than 2 years... Should I add checkver anf autoupdate?

Reply:
There is no reason to not add it. Repository is not archived, no official statement, anything, ...